### PR TITLE
Second page home

### PR DESCRIPTION
**Pull request by paraketsova**
Update PaginationLinks  for userPage and homePage

### CODE REVIEW

File: components/PaginationLinks.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import Link from 'next/link';
+import {useRouter} from "next/router";
```
Comment (clbn) [inline components/PaginationLinks.js]: Please fix the code style here. Also, put it before Link.

File: components/PaginationLinks.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 
-const PaginationLinks = ({ offset, username, hideOnFirst }) => {
-  const normalOffset = +(offset || 0); // normalize offset which can be empty string or undefined
+const PaginationLinks = ({ hideOnFirst, pathname}) => {
```
Comment (clbn) [inline components/PaginationLinks.js]: Let's reverse the parameter order here.

File: components/PaginationLinks.js [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+  const { query: { offset } } = useRouter();
 
+  const normalOffset = +(offset || 0); // normalize offset which can be empty string or undefined
   const newerOffset = Math.max(normalOffset - 30, 0);
   const olderOffset = normalOffset + 30;
 
-  const newerLink = '/' + username + (newerOffset ? '?offset=' + newerOffset : '');
-  const olderLink = '/' + username + '?offset=' + olderOffset;
+  const newerLink = pathname + (newerOffset ? '?offset=' + newerOffset : '');
+  const olderLink = pathname + '?offset=' + olderOffset;
```
Comment (clbn) [inline components/PaginationLinks.js:12]: This won't work if `pathname` has parameters (and `?`) in it, like in Search. But I guess we can leave that fix for future updates, this is good enough for HomePage and UserPage.

File: pages/index.js [hunk 4]
```diff
@@ -5,6 +5,8 @@ import { loadHomePage } from '../store/actions';
 import SignIn from '../components/SignIn';
 import DummyPost from '../components/DummyPost';
 import Post from '../components/Post';
+import PaginationLinks from '../components/PaginationLinks';
+import {useRouter} from "next/router";
```
Comment (clbn) [inline pages/index.js]: Code style. Also, this line should be moved up to the first group of imports.

**Review by clbn — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 4 above)
Comment (paraketsova) [reply pages/index.js]: I suppose I can delete row 9 because it doesn't active now.

**Review by clbn — APPROVED** [review]

**Pull request merged**